projects/testing-library/src/lib/models.ts

```diff
@@ -220,6 +226,24 @@ export interface RenderComponentOptions<ComponentType, Q extends Queries = typeo
    * })
    */
   componentOutputs?: Partial<ComponentType>;
+
+  /**
+   * @description
+   * An object to subscribe to EventEmitters/Observables of the component
+   *
+   * @default
+   * {}
+   *
+   * @example
+   * const sendValue = (value) => { ... }
+   * await render(AppComponent, {
+   *  subscribeToOutputs: {
+   *    send: (_v:any) => void
+   *  }
+   * })
+   */
+  subscribeToOutputs?: SubscribeToOutputsKeysWithCallback<ComponentType>;
```

Comment:
I'm wonder if there's a better name for this. Imho `subscribeToOutputs` doesn't "stick".
Taking a look at Playwright Angular Component Tests, they use `on`, which I like more.
On the other hand... I'm wondering if it's intuitive and clear to new users.

```ts
  const component = await mount(ButtonComponent, {
    on: {
      submit: (data: string) => messages.push(data),
    },
  });
``` 

I wonder if we can just replace `componentOutputs` with the new behavior.
We would've take into account that users can pass an `EventEmitter` or a simple object `{ emit: fn }`, this signature will be deprecated. There are existing test cases within this project that cover this. 

Thoughts ?

Reply:
I completely agree with the name choice. Here are my thoughts:

Unlike `ComponentInputs`, we are not restricting users to only subscribe to actual outputs. Instead, the API allows them to subscribe to any object with a `subscribe` method. We could read the `ComponentMirror` and throw an error if it’s not a declared output, but would this behavior add any value? In my opinion, probably not.

Therefore, using `on` seems like the better option to me. I'm also fine with overloading (checking for `'emit' in v` should be sufficient i guess).

Reply:
> We could read the ComponentMirror and throw an error if it’s not a declared output, but would this behavior add any value? 

I agree that this doesn't add value. If needed we can revisit this later.

> Therefore, using on seems like the better option to me. I'm also fine with overloading (checking for 'emit' in v should be sufficient i guess).

Because of the first comment I assume you're not a fan of "re-using" the `componentOutput` property? I'm fine with using `on` 👍